app/code/Magento/CustomerGraphQl/etc/schema.graphqls

```diff
@@ -28,7 +28,8 @@ input CustomerAddressInput {
     city: String @doc(description: "The city or town")
     region: CustomerAddressRegionInput @doc(description: "An object containing the region name, region code, and region ID")
     postcode: String @doc(description: "The customer's ZIP or postal code")
-    country_id: CountryCodeEnum @doc(description: "The customer's country")
+    country_id: CountryCodeEnum @doc(description: "The customer's country") @deprecated(reason: "Use country_code instead.")
```

Comment:
```suggestion
    country_id: CountryCodeEnum @doc(description: "Deprecated: Use `country_code` instead")
```
Input types do not support `@deprecated` directive.

Reply:
Fixed.